### ISSUE TITLE
Independent Cast Bar option

### ISSUE THREAD

**Issue by slawry00** [Aviana/LunaUnitFrames#355]
Could you make an option for the cast bar to be independent of the rest of the frame when "hidden"? 

When something begins casting, it shrinks the healthbar/powerbar to fit in the original frame size with the cast bar included, but would look much better if it was just a stand-alone. I know this is possible if "hidden" is not checked, but then it is a constant black bar (and also makes the portrait huge and offsets a bunch of indicators).

In other words, I'd just like the castbar to stay hidden, but pop up when needed without it affecting the rest of the frame at all... which I guess only makes sense if it is the top or bottom bar.

Thanks for all that you do with this addon!

**Comment by Aviana** [comment]
The castbar is part of the frame and will stay part of it. If you are not satisfied with it you can always deactivate it and chose something that suits your needs.
